h/assets.yaml

```diff
@@ -195,6 +195,7 @@ inject:
 
 inject_js:
   contents:
+  - url
```

Comment:
The shim already does feature detetion.
See here: https://github.com/hypothesis/h/blob/master/h/static/scripts/vendor/polyfills/url.js#L11